client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,93 @@
+import { Grid, Box, Typography, Rating } from '@mui/material';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import AvatarDisplay from '../../components/AvatarDisplay/AvatarDisplay';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import RequestForm from './RequestForm/RequestForm';
+import useStyles from './useStyles';
+import { useAuth } from '../../context/useAuthContext';
+
+const boxShadow =
+  '0px 0px 1.5px rgba(0, 0, 0, 0.006),0px 0px 3.6px rgba(0, 0, 0, 0.009),0px 0px 6.8px rgba(0, 0, 0, 0.011),0px 0px 12px rgba(0, 0, 0, 0.015),0px 0px 22.1px rgba(0, 0, 0, 0.027),0px 0px 48px rgba(0, 0, 0, 0.1)';
+
+const mockProfile = {
+  name: 'Norma Byers',
+  description: 'Loving pet sitter',
+  location: 'Toronto, Ontario',
+  aboutMe:
+    'Animals are my passion! I will look after your pets with loving care. I have some availability for pet care in my home as well. I have 10 yrs experience at the Animal Hospital, and have owned multiple pets for many years, including numerous rescues. Kindly email, text or call me and I will respond promptly!',
+  payRate: 14,
+  rating: 4,
+};
+
+export default function ProfileDetails(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <PageContainer>
+      <Grid m={window.innerWidth < 600 ? 2 : 4} justifyContent="space-around" container alignItems="flex-start">
```

Comment:
Using `window.innerWidth` is fine but I'm not sure whether it works when we resize? One option would be to store these values in a state

Ref:  https://www.pluralsight.com/guides/render-window-resize-react

Reply:
Ah brilliant that is the exact solution I was looking for when I was troubleshooting it. Thanks!